infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/replication/receive/LogReplicationSinkManager.java

```diff
@@ -79,6 +81,8 @@
     @Getter
     private ObservableValue rxMessageCount = new ObservableValue(rxMessageCounter);
 
+    private AtomicInteger dataConsistent = new AtomicInteger(1);
```

Comment:
Can we have a short comment on this dataConsistent?

Reply:
I have it in initCommonParams() where it first gets initialized but can move it here.